projects/packages/videopress/src/client/block-editor/extensions/index.ts

```diff
@@ -17,6 +20,21 @@ debug( 'Extensions: %o', extensions );
  * @returns {boolean} - Whether the extension is enabled.
  */
 export function isExtensionEnabled( extension: string ) {
-	const vpExtension = extensions.find( ( ext: VideoPressExtensionProp ) => ext.name === extension );
-	return !! vpExtension?.isEnabled;
+	const vpExtension = extensions.find(
+		( ext: VideoPressExtensionProps ) => ext.name === extension
```

Comment:
You could cast it directly at the array level with `const extensions = window?.videoPressExtensions || <VideoPressExtensionsProps>[];` to avoid those two casts on `ext`.

Reply:
it's already done there, I think, is it?

```ts
export type VideoPressExtensionsProps = Array< VideoPressExtensionProps >;
```

Reply:
`window?.videoPressExtensions` is correct, I was thinking about the `|| []` part. Though thinking about it now it is not necessary either. You can just use `const vpExtension = extensions.find( ext => ext.name === extension );` directly and `ext`'s type is correctly inferred.

Reply:
https://github.com/Automattic/jetpack/pull/27179/commits/2081b7bef79107a6649d2b54f80d57135dd9a04a